### DISCUSSION TITLE
New benchmark suggestion: equilibrium shapes, polycrystal, triple junctions

### DISCUSSION THREAD

**Discussion by wd15** [usnistgov/pfhub#1415] [Ideas]
The following is copied and pasted from [Gitter](https://gitter.im/usnistgov/chimad-phase-field?at=6283f23b10cfc315bc83c4ca) from @dudris. Could you possibly expand on your idea in the comments below and possibly provide some references. Thanks.

> Dear Dr. Wheeler,
> are there any thoughts for expanding the list of benchmarks? I have been working on multi-phse field model including inclination dependence of interface energy. When validating my implementation I simulated the Wulff shape in 2-phase system and developed a quantitative methodology applicable to any model simulating the anisotorpic isolated particle. Besides that, I think that there should be a benchmark comparing results of triple junction angles among different models (also worked on that). Would you or somebody else be interested in hearing more?
> Kind regards,
> Martin Minar, PhD student @ KU Leuven, Belgium

**Comment by dudris** [comment]
First of all, thank you for your interest!
In the quoted message I write about two separate things: Wulff shapes and triple junction angles. I should emphasize that I have worked on 2D simulations. 
**Wulff shapes validation**
I was interested in assessing the model performance when varying strength of anisotropy. I benchmarked the accuracy of representing the anisotorpic curvature driving force by the phase field model. 
I assume a 2-phase system (a circle in a matrix) and inclination-dependent interface energy. I used the most common anisotropy function _f(\theta)=1 + \delta\cos(n\theta)_. There is only curvature driving force (i.e. anisotropic Allen-Cahn equation, in my case in the framework of a multi-phase field model by Moelans). The circle shrinks and attains the Wulff shape.  I propose then 2 ways to benchmark the model:
1) **match of the shape.** The phase-field contour can easily be scaled to unit radius for comparison to the analytic Wulff shape. When the two are plotted in polar coordinates, the comparison is a 1D thing (angle vs. radius). I computed sum of squared residuals in radius to characterize the quality of match (normalized by number of points along the perimeter). 
2) **shrinkage rate.** for the anisotropy function above I derived the shrinkage rate (i.e. how fast the area of the particle should disappear due to the anisotorpic curvature driving force) as function of the strength of anisotropy \delta. When the shrinking contour is a Wulff shape, the shrinkage rate is a constant. Whichever model is used, it should converge to that value.
Both benchmarks are analytic cases, independent on the model or numerical methods used. The first one can be more difficult to compare if not everyone uses the same tool to compute the sum of squared residuals, but at least it allows to quantitatively compare results of the same author.

**Comment by dudris** [comment]
**Kinetically compensated Wulff shape**
Then I have a case of combined inclination dependence of interface energy and kinetic coefficient. The kinetic anisotropy is chosen such to ompensate the anisotorpic curvature driving force, resulting in seemingly isotropic behavior. The initial circle should remain circular and should shrink as with isotropic interface energy. The match of the contour to the circle and shrinkage rate was assessed in the same spirit like above. This validates that the model is able to implement the kinetic and IE anisotorpy independently.

**Comment by dudris** [comment]
**Triple junction angles**
I measured angles in pair-wise isotorpic triple junction angles. It is a nice benchmark because it represents a super important piece of physics (the force balance in triple junctions) and also it allows to observe the ghost phases, if they are present. The inclination-dependent interface s in a triple junction is a different story that I have not investigated yet. Anmyway, I think that it would be useful to have a well defined methodology against whih one might ompare the performane of their implementation angainst the Young's equaition giving the angles...

**Comment by dudris** [comment]
I have a [preprint ](https://arxiv.org/abs/2201.00922) out, providing a lot of details about how exatly I was doing things. I should say, though,  that I am now rewriting the paper because the reviewers lacked the context. The second version will emphasize different points about my work, but I will not do more simulations for that paper.

**Comment by wd15** [comment]
Hi @dudris, I'm going to try and get some other members of the group to comment on this. Overall, I think that some sort of polycrystal model would be a good idea. Do we need a polycrystal model that's just isotropic to start or is that too boring? Also, can we define it so that a variety of approaches would work, not just multi-phase field (i.e. orientation equation). Alternatively a benchmark that exclusively focuses on triple junction behavior.

**Reply by dudris** [reply]
I think that in practical terms, any reliable multi-phase field method (or any model mimicking curvature driving force) should correctly reproduce
1) triple juntction angles with pair-wise isotropic interfaces
2) isotropic curvature driving force (shrinking circle or circles)
   a) kinetics
   b) shape is a circle
3) anisotropic curvature driving force
   a) kinetics
   b) Wulff shape
4) triple junction angles with anisotropic interfaces - I don't have validation concept for this one yet

None of these should be taken for granted and all should be validated in all physially relevant cases (thorough parameter studies). It will be extreme conditions where the models will behave differently the most I guess...

The approaches I have in the preprint and briefly desribe above only assume curvature driving fore, ie they are model-independent..
